endaq/calc/utils.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import Optional
+import warnings
+
+import numpy as np
+import pandas as pd
+
+
+def logfreqs(
+    df: pd.DateTime, init_freq: Optional[float] = None, bins_per_octave: float = 12
```

Comment:
this takes in the whole dataframe; should I have it only take in an index? It doesn't use the rest of the data really

Reply:
Since everything else takes the whole dataframe, I think it makes sense to keep it consistent.  It's just passing references anyway, so.

Reply:
Is the type hint correct? Should it be `Dataframe`?

Reply:
...yes =_= this is proof that reviews work, thank you